src/cli/util.js

```diff
@@ -54,7 +54,7 @@ function diff(a, b) {
 
 function handleError(context, filename, error) {
   if (error instanceof errors.UndefinedParserError) {
-    if (context.argv["write"] && process.stdout.isTTY) {
+    if (context.argv["write"] && process.stdout.isTTY && !thirdParty.isCI()) {
```

Comment:
Since `process.stdout.isTTY` and `!thirdParty.isCI()` are always used together, it'd be better to extract them to be a new function.

Reply:
Sounds like a reasonable idea, but how would you call it? I could not come up with a good name.

Reply:
How about calling it `isTTY` directly?

Reply:
Yes – calling `isTTY` plus adding a comment explaining it would be nice